workflows/data_pipelines/bilans_financiers/processor.py

```diff
@@ -98,6 +98,7 @@ def preprocess_data(self) -> None:
             index=False,
         )
 
-        DataProcessor.push_unique_count(
-            df_bilan.siren, Notification.notification_xcom_key
+        DataProcessor.push_message(
```

Comment:
Here for instance no description is given. So the push message will only contain the count.